rest-service/manager_rest/rest/resources_v1/events.py

```diff
@@ -51,38 +54,98 @@ class Events(SecuredResource):
     DEFAULT_SEARCH_SIZE = 10000
 
     @staticmethod
-    def _build_select_query(_include, filters, pagination, sort):
+    def _apply_range_filters(query, model, range_filters):
+        """Apply range filters to query.
+
+        :param query: Query in which the filtering should be applied
+        :type query: :class:`sqlalchemy.orm.query.Query`
+        :param model: Model to use to apply the filtering
+        :type model:
+            :class:`manager_rest.storage.resource_models.Event`
+            :class:`manager_rest.storage.resource_models.Log`
+        :param range_filters: Range filters passed as a request argument
+        :type range_filters: dict(str, dict(str))
+        :returns: Query with filtering applied
+        :rtype: :class:`sqlalchemy.orm.query.Query`
+
+        """
+        for field, range_filter in range_filters:
+            # Drop `@` prefix for compatibility
+            # with old Elasticsearch based implementation
+            field = field.lstrip('@')
+            if hasattr(model, field):
+                query = Events._apply_range_filters(
+                    query, model, field, range_filter)
+        return query
+
+    @staticmethod
+    def _apply_range_filter(query, model, field, range_filter):
+        """Apply a range filter to query.
+
+        :param query: Query in which the filtering should be applied
+        :type query: :class:`sqlalchemy.orm.query.Query`
+        :param model: Model to use to apply the filtering
+        :type model:
+            :class:`manager_rest.storage.resource_models.Event`
+            :class:`manager_rest.storage.resource_models.Log`
+        :param field: Field in the model that should be filtered
+        :type field: str
+        :param range_filter: Range filter passed as a request argument
+        :type range_filter: dict(str)
+        :returns: Query with filtering applied
+        :rtype: :class:`sqlalchemy.orm.query.Query`
+
+        """
+        if 'from' in range_filter:
```

Comment:
what's the behavior if 'from' or 'to' are not in range_filter? what if both of them are not there?